src/app/submission/components/export-to-archives/export-to-archives.component.ts

```diff
@@ -0,0 +1,190 @@
+import {Component, Input, OnInit} from '@angular/core';
```

Comment:
Why is this file new here? It seems to be doing a lot more than just SCEA conversion

Reply:
Ah now I see that there is a bit of refactoring! Nice! Just mention this in the PR description